app-emulation/docker/Manifest

```diff
@@ -1 +1 @@
-DIST docker-19.03.1.tar.gz 16652136 BLAKE2B be5c001e48bdce0ea8e520c131ff1699d12e39c45cf7dba3452689879e29abe12509c5d70e4a82eb66ac599b85717cd7e057cff1a41ba552eb2ce549e692d991 SHA512 92b4e5fe2bbf96a261d290ca807550af45146be9d21680940bd6aa45d9127ae8ddbc706df4056f1720ed6975a2a92004f1789fae4109c50206904ad827d4bf2e
+DIST docker-19.03.2.tar.gz 16657356 BLAKE2B 3e48ea67f1ef65786c30b4d47b29dbe92d7fb7b043d2c4f15c07cdc929554a07870ce5ade2c1fe26eebea74d28dc71752e9c96e86fe12e0d96608059b5c91f11 SHA512 6b594fdbb53dcc0228781375a3884eb370446738c44f7c1e42945c4ccc263e75f53d984bc8ea6a6a498446859e667305bd967299c12956f1cb925d868a4bf2b8
```

Comment:
Since this changed, maybe `DOCKER_GITCOMMIT="…"` needs to change in `app-emulation/docker/docker-9999.ebuild` as well? Haven't tested it yet, but am asking in order to understand the last issue.

Reply:
Ah, actually it's different now. Still I wonder what this commit ID is about.

Reply:
This is a little too specific to docker. Anyway...

The ebuilds fetches a tarball docker-ce 19.03.x, extracts, and builds under `components/engine`.
Then it runs `components/engine/hack/make.sh`, which clones again https://github.com/docker/docker.
Then it checks if the given `DOCKER_GITCOMMIT` exists in the main docker git repo.

See https://github.com/docker/docker-ce/blob/adf26ee1003dea6865036b99aff727e502bf0fb4/components/engine/hack/make.sh#L46

Reply:
The variable should be updated because it's coming up here in `Git commit:        d7080c1` which points to 18.06 https://github.com/docker/docker-ce/commit/d7080c1 instead of 19.03:
```
core@localhost ~ $ docker version
Client:
 Version:           19.03.1
 API version:       1.40
 Go version:        go1.12.9
 Git commit:        d7080c1
 Built:             Thu Jul 25 20:38:41 2019
 OS/Arch:           linux/amd64
 Experimental:      false
```
I hope it has no other impact except misinformation…

Reply:
It should be: `DOCKER_GITCOMMIT="6a30dfc"` in line 22

Reply:
Updated `DOCKER_GITCOMMIT`.